command/import.go

```diff
@@ -168,6 +172,34 @@ func runImport(cmd *Command, args []string) {
 				fmt.Printf("%s\n\tstatus: %s\n\tid=%s\n", success.FullName, verb, success.Id)
 			}
 		}
+		fmt.Printf("\nTest Successes - %d\n", len(testSuccesses))
```

Comment:
Should we be looking to see if there are any successes before outputting the Header?

Reply:
I think it should show zero if there are no successes.  Do you mean to not show Test Successes and Test Failures if no tests were run?

Reply:
No, I think you are on the right track.  I prefer consistency over conciseness (sp?).